hw/top_darjeeling/ip_autogen/pinmux/data/pinmux.hjson

```diff
@@ -260,19 +260,25 @@
                   regwen:       "MIO_PAD_ATTR_REGWEN",
                   regwen_multi: "true",
                   cname:        "MIO_PAD",
-                  resval:       0
                   fields: [
                     { bits: "0",
                       name: "invert",
                       desc: "Invert input and output levels."
+                      resval: 0
                     },
                     { bits: "1",
                       name: "virtual_od_en",
                       desc: "Enable virtual open drain."
+                      resval: 0
                     },
                     { bits: "2",
                       name: "pull_en",
                       desc: "Enable pull-up or pull-down resistor."
+                      // Note: The special index (30) where this has a different resval depends on
+                      // the TargetCfg parameter that gets passed in. That, in turn, is configured
+                      // at chip level and it would be nicer if we could resolve it as we expanded
+                      // this template.
+                      resval: (multireg_idx == 30)
```

Comment:
Does this work for Darjeeling? I'm asking because for Earlgrey we have
https://github.com/lowRISC/opentitan/blob/6289ededa021225d7d04157a64e4bf022e63da76/hw/top_earlgrey/rtl/autogen/chip_earlgrey_asic.sv#L99
but for Darjeeling it is
https://github.com/lowRISC/opentitan/blob/6289ededa021225d7d04157a64e4bf022e63da76/hw/top_darjeeling/rtl/autogen/chip_darjeeling_asic.sv#L122

CC @Razer6

Reply:
I think the version I just force-pushed should sort this out.